pandas/tests/tseries/offsets/conftest.py

```diff
@@ -14,11 +64,25 @@ def offset_types(request):
 @pytest.fixture(
     params=[
         getattr(offsets, o)
-        for o in offsets.__all__
+        for o in DATE_OFFSETS
+        if issubclass(getattr(offsets, o), offsets.MonthOffset) and o != "MonthOffset"
+    ]
+)
+def date_month_classes(request):
+    """
+    Fixture for month based datetime offsets available for a time series.
+    """
+    return request.param
+
+
+@pytest.fixture(
+    params=[
+        getattr(offsets, o)
+        for o in BUSINESS_OFFSETS
         if issubclass(getattr(offsets, o), offsets.MonthOffset) and o != "MonthOffset"
     ]
 )
-def month_classes(request):
+def business_month_classes(request):
     """
     Fixture for month based datetime offsets available for a time series.
```

Comment:
same docstring for business_month_classes as date_month_classes. should differentiate.

Reply:
Added in #31034